### PR TITLE
Update ar_LB.all.json

### PR DESCRIPTION
**Pull request by additiverich**
Fix a broken translation for Arabic

### CODE REVIEW

**Review by l0ud0gg — APPROVED** [review]

**Comment by additiverich** [comment]
@l0ud0gg Some other translations still had the English "day"

**Comment by additiverich** [comment]
And the English and es_MX translations had some intervals with automatic plurals which we don't need anymore

**Comment by additiverich** [comment]
And es_MX had some renewal information in the trial period translations

File: site/ar_LB.all.json [hunk 1]
```diff
@@ -1202,7 +1202,7 @@
   },
   "shopping_info_trial_period_offer": {
     "one": "جرب الإصدار التجريبي المجاني لمدة 24 ساعة الآن!",
-    "other": "{{.Count}} التجريبي المجاني من {{.Count}} day!"
+    "other": "جرب الإصدار التجريبي المجاني لمدة {{.Count}} أيام!"
```
Comment (PeculiarGoat) [inline site/ar_LB.all.json:1205]: These should also have all pluralities available to the language in question :/

Comment (PeculiarGoat) [reply site/ar_LB.all.json:1209]: @additiverich shouldn't `two` to `other` all have the `{{.Count}}` variable?

File: site/hr_HR.all.json [hunk 2]
```diff
@@ -1119,7 +1119,7 @@
   },
   "shopping_info_trial_period_offer": {
     "one": "Isprobajte svoju besplatnu probnu verziju od 24 sata sada!",
-    "other": "Isprobajte svoju besplatnu {{.Count}} day!"
+    "other": "Isprobajte {{.Count}}-dnevno besplatno probno razdoblje!"
```
Comment (PeculiarGoat) [inline site/hr_HR.all.json:1122]: This language has `few` too.

**Review by PeculiarGoat — APPROVED** [review]
👍

**Review by l0ud0gg — APPROVED** [review]
nice 👍 we might need to update the db translations if they havent already

**Pull request merged**